src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -190,6 +186,16 @@ namespace AppInstaller::Repository
                 details.Data = s_Source_WingetCommunityDefault_Data;
                 result.emplace_back(std::move(details));
             }
+
+            if (Settings::ExperimentalFeature::IsEnabled(Settings::ExperimentalFeature::Feature::ExperimentalMSStore))

```

Comment:
>if [](start = 12, length = 2)

Put the block inside the above one so it is more clear that they are part of the same case.